bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/DiscoveryResultBuilder.java

```diff
@@ -134,7 +134,7 @@ public DiscoveryResultBuilder withTTL(long ttl) {
      */
     public DiscoveryResult build() {
         return new DiscoveryResultImpl(thingTypeUID, thingUID, bridgeUID, properties, representationProperty, label,
-                ttl);
+                ttl, null);
```

Comment:
I agree with @sjka that we must not require users of the Builder to add a new method call everywhere, but maybe it can be derived automatically (or stay null, if we say that it is not a mandatory property)?